src/tools/compiletest/src/main.rs

```diff
@@ -517,7 +517,7 @@ pub fn make_test_name(config: &Config, testpaths: &TestPaths) -> test::TestName
     //
     //    run-pass/foo/bar/baz.rs
     let path =
-        PathBuf::from(config.mode.to_string())
+        PathBuf::from(config.src_base.file_name().unwrap());
```

Comment:
`config.src_base` is already a `PathBuf` so wouldn't it suffice to just write `config.src_base`?

Reply:
That will print the absolute path to the test. I wanted to make it relative to the current working directory, and this is the crude approximation I ended up with.

Reply:
Ah OK